refreshed spin operator readme (#143)
spinkube/spin-operator: README.md

To get started, check out the [Spin Operator quickstart](http://spinkube.dev/docs/spin-operator/quickstart/)

![Conceptual Overview Diagram of Spin Operator](/static/image/spin-operator-diagram.png)

## Documentation

To learn more about the Spin Operator and the SpinKube organization, please visit [the official SpinKube documentation](https://spinkube.dev). 

## Feedback

The remaining articles are under construction. You're welcome to view and open both [Spin Operator](https://github.com/spinkube/spin-operator/issues) and [documentation](https://github.com/spinkube/documentation/issues) issues and feature requests. As this work is under development, please note that current features, functionality and supporting documentation are likely to change as the projects evolve and improvements are made.

For questions or support, please visit our [Slack channel](TODO).

## Contributing (Documentation)

> If you would like to contribute to SpinKube and Spin Operator, please visit this [documentation contributing](http://spinkube.dev/docs/contribution-guidelines/) page.

Reviewer: ```suggestion
If you would like to contribute to SpinKube and Spin Operator, please visit the [contribution guide](http://spinkube.dev/docs/contribution-guidelines/).
```
